addons/survey/models/survey_question.py

```diff
@@ -57,6 +57,7 @@ def default_get(self, fields):
     description = fields.Html(
         'Description', translate=True, sanitize=False,  # TDE TODO: sanitize but find a way to keep youtube iframe media stuff
         help="Use this field to add additional explanations about your question or to illustrate it with pictures or a video")
+    placeholder = fields.Char("Placeholder", translate=True, compute="_compute_placeholder", store=True, readonly=False)
```

Comment:
Would rename to "question_placeholder" to help greping, just placeholder will be very annoying to grep.